### PR TITLE
DR-750 Add back unit test suite

### PR DESCRIPTION
**Pull request by Aish1990**
**Steps:** 

- [x] Update the current unit tests to make them run (and pass)
- [ ] Add CI integration to run unit tests of every Pull Request
- [ ] Add code-coverage checks to the CI
- [ ] Add more unit tests to achieve higher code coverage

**Definition of done:** Every Pull-Request should check that the unit tests are passing. The unit tests should achieve a high code coverage (70% or more).

### CODE REVIEW

**Comment by dmohns** [comment]
Hey @Aish1990 I updated the PR description to be more clear.

Could you address the failing linters?

**Comment by dmohns** [comment]
Hey @Aish1990 you have syntax errors in the YAML files. Could you please fix them?

File: apps/drec-api/src/index.spec.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
```
Comment (dmohns) [inline apps/drec-api/src/index.spec.ts]: Please don't use file-wide `eslint-disable`. If you need to disable it, please do so for the line that cannot be fixed.

Comment (Aish1990) [reply apps/drec-api/src/index.spec.ts]: Sir, actually it's like left empty intentionally...  
```   
  IsISO31661Alpha2: jest.fn(() => (target: any, propertyKey: string) => {}),
  IsString: jest.fn(() => (target: any, propertyKey: string) => {}),
  IsEmail: jest.fn(() => (target: any, propertyKey: string) => {}),
  IsOptional: jest.fn(() => (target: any, propertyKey: string) => {}),
  IsArray: jest.fn(() => (target: any, propertyKey: string) => {}),
  IsNotEmpty: jest.fn(() => (target: any, propertyKey: string) => {}),
```

Comment (dmohns) [reply apps/drec-api/src/index.spec.ts]: I understand. But I'm talking about file-wide ignore.

Please use

```js
jest.mock('class-validator', () => ({
  // eslint-disable-next-line no-empty-function
  IsISO31661Alpha2: jest.fn(() => (target: any, propertyKey: string) => {}),
  
  // eslint-disable-next-line no-empty-function
  IsString: jest.fn(() => (target: any, propertyKey: string) => {}),
  
  // eslint-disable-next-line no-empty-function
  IsEmail: jest.fn(() => (target: any, propertyKey: string) => {}),
  
  // eslint-disable-next-line no-empty-function
  IsOptional: jest.fn(() => (target: any, propertyKey: string) => {}),
  
  // eslint-disable-next-line no-empty-function
  IsArray: jest.fn(() => (target: any, propertyKey: string) => {}),
  
  // eslint-disable-next-line no-empty-function
  IsNotEmpty: jest.fn(() => (target: any, propertyKey: string) => {}),
}));
```

instead

Comment (Aish1990) [reply apps/drec-api/src/index.spec.ts]: Okay sir...

**Review by musayann — APPROVED** [review]

**Review by musayann — APPROVED** [review]

**Pull request merged**